README.md

```diff
@@ -86,10 +74,20 @@ Or to specify the location explicitly:
 `GOVUK_CONTENT_SCHEMAS_PATH=/some/dir/govuk-content-schemas bundle exec rake`
 
 #### Debugging Integration tests
+
 If you want to see the page that is being tested in our integration tests, you can use
 `save_and_open_page` to see what's rendered. This is helpful when a page is mostly comprised of
 GOV.UK Publishing Components
 
+### Components
+
+Pages are rendered using components. Components can be specific to government-frontend or shared between applications.
+
+Components specific to government-frontend are [within the application](https://github.com/alphagov/government-frontend/tree/master/app/views/components) and follow rules set out by the [govuk_publishing_components](https://github.com/alphagov/govuk_publishing_components) gem. They are documented in the [government-frontend component guide](https://government-frontend.herokuapp.com/component-guide).
+
+Configuration for the navigation links shown on the B variant of the ContentPagesNav A/B test is covered [separately](docs/navigation-links.md)
```

Comment:
This links to a 404 and seems unrelated to components - I'd suggest removing.

Reply:
Good spot, I've removed it in https://github.com/alphagov/government-frontend/pull/2108/commits/a79d31dccf199d58096c0dccd744ce3ef616be67.